sdk/devtestlabs/Azure.ResourceManager.DevTestLabs/src/autorest.md

```diff
@@ -49,4 +52,136 @@ rename-rules:
+  User: DevTestLabUser
+  VirtualNetwork: DevTestLabVirtualNetwork
+  ApplicableSchedule: DevTestLabApplicableSchedule
+  ApplyArtifactsRequest: DevTestLabVmApplyArtifactsContent
+  ArmTemplateInfo: DevTestLabArmTemplateInfo
+  ArmTemplateParameterProperties: DevTestLabArmTemplateParameter
+  ArtifactDeploymentStatusProperties: DevTestLabArtifactDeploymentStatus
+  ArtifactInstallProperties: DevTestLabArtifactInstallInfo
+  ArtifactParameterProperties: DevTestLabArtifactParameter
+  AttachDiskProperties: DevTestLabDiskAttachContent
+  AttachDiskProperties.leasedByLabVmId: -|arm-id
+  AttachNewDataDiskOptions: AttachNewDataDiskDetails
+  TargetCostProperties: DevTestLabTargetCost
+  CostThresholdProperties: DevTestLabCostThreshold
+  CostThresholdStatus: DevTestLabCostThresholdStatus
+  CostType: DevTestLabCostType
+  LabCostDetailsProperties: DevTestLabCostDetails
+  CustomImageOsType: DevTestLabCustomImageOsType
+  CustomImagePropertiesCustom: DevTestLabCustomImageVhd
+  CustomImagePropertiesFromVm: DevTestLabCustomImageVm
+  CustomImagePropertiesFromPlan: DevTestLabCustomImagePlan
+  DataDiskStorageTypeInfo: DevTestLabDataDiskStorageTypeInfo
+  StorageType: DevTestLabStorageType
+  DataDiskProperties: DevTestLabDataDiskProperties
+  DataDiskProperties.existingLabDiskId: -|arm-id
+  DetachDataDiskProperties: DevTestLabVmDetachDataDiskContent
+  DetachDiskProperties: DevTestLabDiskDetachContent
+  DetachDiskProperties.leasedByLabVmId: -|arm-id
+  UpdateResource: DevTestLabResourcePatch
+  EnableStatus: DevTestLabEnableStatus
+  EnvironmentDeploymentProperties: DevTestLabEnvironmentDeployment
+  EnvironmentDeploymentProperties.armTemplateId: -|arm-id
+  EnvironmentPermission: DevTestLabEnvironmentPermission
+  EvaluatePoliciesRequest: DevTestLabEvaluatePoliciesContent
+  EvaluatePoliciesProperties: DevTestLabEvaluatePolicy
+  EvaluatePoliciesResponse: DevTestLabEvaluatePoliciesResult
+  PolicySetResult: DevTestLabPolicySetResult
+  Event: DevTestLabNotificationChannelEvent
+  NotificationChannelEventType: DevTestLabNotificationChannelEventType
+  ExportResourceUsageParameters: DevTestLabExportResourceUsageContent
+  ExternalSubnet: DevTestLabExternalSubnet
+  ExternalSubnet.id: -|arm-id
+  SubnetOverride: DevTestLabSubnetOverride
+  SubnetOverride.resourceId: -|arm-id
+  FileUploadOptions: DevTestLabFileUploadOption
+  GalleryImage: DevTestLabGalleryImage
+  GalleryImage.properties.enabled: IsEnabled
+  GalleryImageReference: DevTestLabGalleryImageReference
+  GenerateArmTemplateRequest: DevTestLabArtifactGenerateArmTemplateContent
+  ParameterInfo: DevTestLabParameter
+  GenerateUploadUriParameter: DevTestLabGenerateUploadUriContent
+  GenerateUploadUriResponse: DevTestLabGenerateUploadUriResult
+  HostCachingOptions: DevTestLabHostCachingOption
+  IdentityProperties: DevTestLabManagedIdentity
+  IdentityProperties.principalId: -|uuid
+  ImportLabVirtualMachineRequest: DevTestLabImportVirtualMachineContent
```

Comment:
As you  changed the `LabVirtualMachine` to `DevTestLabVm`, so would this be `DevTestLabImportVmContent`